torch/_inductor/fx_utils.py

```diff
@@ -217,12 +262,21 @@ def should_process_node(node):
             ):
                 continue
 
+            # If this is a subgraph node, and we're here, then the subgraph update above
+            # found new nodes.  Any users of this subgraph also need to update, but we
+            # can otherwise short-circuit the process at this point.
+            if self._is_subgraph_node(node):
+                to_process.update(id(user) for user in node.users)
+                self.processed_hashes.add(self.hash_node(node))
+                continue
```

Comment:
Why don't we update the FakeTensors on this node?

Reply:
This node is not a callable, but a `get_attr` node that resolves to a subgraph. We handle it specially so that we force downstream calls to `invoke_subgraph` where needed.